src/config/env.js

```diff
@@ -40,3 +40,5 @@ export const getEnv = (name, defaultValue) => {
 
   throw new Error(`Missing environment variable: ${name}`);
 };
+
+export const APP_TITLE = getEnv("APP_TITLE", "S-Pipes UI");
```

Comment:
please change to `SPipes Editor`


note, we always use only variants: 
SPipes or s-pipes but never S-Pipes

Reply:
done